tests/grid/grid_in_ucd_01.cc

```diff
@@ -0,0 +1,80 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2002 - 2015 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// read in a 2d file in the UNV format used by the Salome program
+
+// the meshes used here were generated by Valentin Zingan using the Salome
+// mesh generator
```

Comment:
Can you also please update the comment here to describe what the test does? I think this is a leftover of the test you copied from...